src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,20 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_in_process(extra_env):
```

Comment:
The current interface for mutator plugins is unfinished but I'd rather radmsa reused it (extending it if needed to support a plugin that could be applied to any fuzzer) rather than added a new code path that only applied to radamsa.
@mbarbella-chromium WDYT?
You could change the naming scheme used for plugins to support wildcards (`*`). As is this has a problem where it will break mac/msan binaries because it will give them a shared object only asan binaries can use.

Reply:
Ping

Reply:
I'm having trouble picturing exactly what reusing the existing interface would look like here. I also don't understand the point about wildcards, could you elaborate?

Either way I do agree that reusing the existing interface would be preferable if we can.

Reply:
I mean the mutator plugins have a hacky naming scheme for indicating that a given plugin should be used with binaries for a certain platform, arch, sanitizer. This doesn't do that I assume it will therefore break (at least) i386 and msan binaries.